test/hotspot/gtest/nmt/test_nmtpreinit.cpp

```diff
@@ -112,7 +113,7 @@ class TestAllocations {
     log_state();
   }
   void test_post() {
-    assert(NMTPreInit::in_preinit_phase() == false,
+    assert(!MemTracker::is_initialized() == false,
```

Comment:
Hmm, I am confused by the double negation :-)

Reply:
:-) Right, I'll fix it